modules/cluster-logging-deploy-clo-cli.adoc

```diff
@@ -82,11 +82,13 @@ metadata:
 spec:
   channel: "4.3" <2>
   name: cluster-logging 
-  source: redhat-operators 
+  source: redhat-operators <3>
   sourceNamespace: openshift-marketplace
 ----
 <1> You must specify the `openshift-logging` Namespace.
 <2> Specify `4.3` as the channel.
+<3> Specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, 
```

Comment:
why should this be different then `openshift-operators-redhat`?  The fact the environment is disconnected seems immaterial.  The namespace `openshift-operators-redhat` was chosen as an alternate to the original global namesapce of 'openshift-operators' so the prometheus stack could be enabled to scrape metrics for global redhat operators

Reply:
This is the catalog source in the Subscription object. In a disconnected install, the source is different than a connected install and the ConfigSource object points to that source. Thus, the cluster logging Subscription must point to the ConfigSource object to get to the source.

**Build the Operator catalog image:**
https://docs.openshift.com/container-platform/4.3/operators/olm-restricted-networks.html#olm-building-operator-catalog-image_olm-restricted-networks
**Configuring OperatorHub for restricted networks**
https://docs.openshift.com/container-platform/4.3/operators/olm-restricted-networks.html#olm-restricted-networks-operatorhub_olm-restricted-networks